### PR TITLE
feature/51-add-csv-examples

### PR DESCRIPTION
**Pull request by afsmythe**
This PR adds sample data to the CSV examples already present. Also, two header columns needed correcting (`ballot_measure_contest` and `candidate_contest`).

### CODE REVIEW

File: docs/csv/example_files/contest.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation
\ No newline at end of file
+id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation
+con001,SE-1,"bs001 bs002”,,Governor of Virginia,ed001,all registered voters,fips,,23,false,Gov,,,
```
Comment (lbirdpew) [inline docs/csv/example_files/contest.txt]: I know this is a dummy file, but maybe change the FIPS code to 51 for VA

File: docs/csv/example_files/ballot_selection.txt [hunk 2]
```diff
@@ -1 +1,6 @@
-id,sequence_order
\ No newline at end of file
+id,sequence_order
```
Comment (jdmgoogle) [inline docs/csv/example_files/ballot_selection.txt]: Ballot selection should be an abstract base class and not have any information associated with it itself. What particular use case did you see for having this be a separate file/class?

File: docs/csv/example_files/ballot_measure_contest.txt [hunk 3]
```diff
@@ -1 +1,2 @@
-id,abbreviation,Contest,ballot_measure_selection_ids,ballot_sub_title,ballot_title,elecoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type
\ No newline at end of file
+id,abbreviation,Contest,ballot_selection_ids,ballot_sub_title,ballot_title,elecoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type
+bmc0001,HB2,bs001 bs002 bs003,Raising levy for School Bond,School Bond Issue,ed001,all registered voters,,54,false,School Bond,42,majority,,This is no good.,No effect,A measure to do raise funds for etc etc,www.ballotmeasure.com,two-thirds,Everything will be great.,It’s a referendum about school funding,referendum,
```
Comment (jdmgoogle) [inline docs/csv/example_files/ballot_measure_contest.txt]: Do you envision IDREFS/other repeated fields always using a space as a separator (e.g., the "bs001 bs002 bs003" field under ballot_selection_ids here)?

File: docs/csv/example_files/ballot_style.txt [hunk 4]
```diff
@@ -1 +1,3 @@
-id,image_uri,ordered_contest_ids,party_ids
\ No newline at end of file
+id,image_uri,ordered_contest_ids,party_ids
+bs00010,http://i.giphy.com/26BoCh3PgT8ai45ji.gif,oc2025,par02
```
Comment (jdmgoogle) [inline docs/csv/example_files/ballot_style.txt]: Could you provide an example in the docs here of a ballot style with multiple OrderedContest elements? Each ballot style here appears to have only one contest each.

File: docs/csv/example_files/candidate_contest.txt [hunk 5]
```diff
@@ -1 +1,3 @@
-id,abbreviation,candidate_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,number_elected,office_ids,primary_party_ids,votes_allowed
\ No newline at end of file
+id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,number_elected,office_ids,primary_party_ids,votes_allowed
+cancon001,SE-1,"bs001 bs002”,,Governor of Virginia,ed001,all registered voters,fips,,49,true,Governor,1,,,1,"off001","par01”,1
```
Comment (jdmgoogle) [inline docs/csv/example_files/candidate_contest.txt]: Some elements are quoted, others aren't. Is there a particular rhyme or reason to that?

File: docs/csv/example_files/contest.txt [hunk 6]
```diff
@@ -1 +1,3 @@
-id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation
\ No newline at end of file
+id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation
```
Comment (jdmgoogle) [inline docs/csv/example_files/contest.txt]: Same question here as for ballot_selection.txt ... contest is an abstract base class and should never be instantiated on its own.

File: docs/csv/example_files/election.txt [hunk 7]
```diff
@@ -1 +1,2 @@
-id,date,name,election_type,state_id,is_statewide,registration_info,absentee_ballot_info,results_uri,polling_hours,has_election_day_registration,registration_deadline,absentee_request_deadline,hours_open_id
\ No newline at end of file
+id,date,name,election_type,state_id,is_statewide,registration_info,absentee_ballot_info,results_uri,polling_hours,has_election_day_registration,registration_deadline,absentee_request_deadline,hours_open_id
+e001,10/08/2016,Best Hot Dog,State,st51,true,www.registrationinfo.com,You can vote absentee,http://hotdogcontest.gov/results,Noon to 3p.m.,true,10/08/2016,,ho002
```
Comment (jdmgoogle) [inline docs/csv/example_files/election.txt]: I won't tell NY that you think VA is going to try and determine the Best Hot Dog all on its own. :)

File: docs/csv/example_files/electoral_district.txt [hunk 8]
```diff
@@ -1 +1,4 @@
-id,external_identifier_type,external_identifier_othertype,external_identifier_value,name,number,type,other_type
\ No newline at end of file
+id,external_identifier_type,external_identifier_othertype,external_identifier_value,name,number,type,other_type
+ed001,ocd-id,,ocd-division/country:us/state:ny/borough:brooklyn,Brooklyn,1,borough,
+ed002,other,community-board,4,CB 4,2,other,community-board
+ed003,county,,4,County Office 4,2,other,county
```
Comment (jdmgoogle) [inline docs/csv/example_files/electoral_district.txt]: Is `county` a valid ExternalIdentifierType?

File: docs/csv/example_files/office.txt [hunk 9]
```diff
@@ -1 +1,4 @@
-id,electoral_district_id,external_identifier_type,external_identifier_othertype,external_identifier_value,filing_deadline,is_partisan,name,office_holder_person_ids,term_type,term_start_date,term_end_date
\ No newline at end of file
+id,electoral_district_id,external_identifier_type,external_identifier_othertype,external_identifier_value,filing_deadline,is_partisan,name,office_holder_person_ids,term_type,term_start_date,term_end_date
+off001,ed001,,,,,true,Deputy Chief of Staff,per50003,full-term,2002-01-21,
+off002,ed001,,,,,true,Deputy Deputy Chief of Staff,per50001,unexpired-term,2002-01-21,
+off003,ed001,,,,,false,General Secretary of Secretaries,per50004,full-term,1984-0101,
```
Comment (jdmgoogle) [inline docs/csv/example_files/office.txt]: Invalid date: 1984-0101

File: docs/csv/example_files/precinct.txt [hunk 10]
```diff
@@ -1 +1,4 @@
-id,ballot_style_id,electoral_district_ids,external_identifier_type,external_identifier_othertype,external_identifier_value,is_mail_only,locality_id,name,number,polling_location_ids,precinct_split_name,ward
\ No newline at end of file
+id,ballot_style_id,electoral_district_ids,external_identifier_type,external_identifier_othertype,external_identifier_value,is_mail_only,locality_id,name,number,polling_location_ids,precinct_split_name,ward
+pre90111,bs00010,"ed001",ocd-id,,"ocd-division/country:us",false,loc001,203 - GEORGETOWN,0203,"poll001 poll002",split13,5
+pre90112,bs00011,"ed002",fips,,42,false,loc001,203 - GEORGETOWN,0203,"poll003",split26,6
```
Comment (jdmgoogle) [inline docs/csv/example_files/precinct.txt]: The state of PA is a polling location for elections being held in Denver, CO? :)

File: docs/csv/example_files/ballot_selection.txt (hunk 2 above)
Comment (afsmythe) [inline docs/csv/example_files/ballot_selection.txt]: Correct. This file is not used in the finished documentation. Will remove the file.

File: docs/csv/example_files/ballot_measure_contest.txt (hunk 3 above)
Comment (afsmythe) [inline docs/csv/example_files/ballot_measure_contest.txt]: Yes - space separated IDREFS are presently provided to meet the VIP specification during the CSV to XML transformation. I'm not aware of any issues with this practice.

File: docs/csv/example_files/ballot_style.txt (hunk 4 above)
Comment (afsmythe) [inline docs/csv/example_files/ballot_style.txt]: Yes - will build this in to the example.

File: docs/csv/example_files/contest.txt (hunk 6 above)
Comment (afsmythe) [inline docs/csv/example_files/contest.txt]: Same answer - this file is not used in the finished documentation. Will remove from the PR.

File: docs/csv/example_files/candidate_contest.txt (hunk 5 above)
Comment (afsmythe) [inline docs/csv/example_files/candidate_contest.txt]: No reason. The CSVs support quoted and un-quoted. I understand wanting to show uniformity, but either example is supported.

File: docs/csv/example_files/electoral_district.txt (hunk 8 above)
Comment (afsmythe) [inline docs/csv/example_files/electoral_district.txt]: No. I'll correct.

File: docs/csv/example_files/ballot_measure_contest.txt (hunk 3 above)
Comment (jdmgoogle) [inline docs/csv/example_files/ballot_measure_contest.txt]: Is this documented somewhere? I may have missed it in my quick skim.

File: docs/csv/example_files/ballot_selection.txt (hunk 2 above)
Comment (jdmgoogle) [inline docs/csv/example_files/ballot_selection.txt]: 👍

File: docs/csv/example_files/source.txt [hunk 11]
```diff
@@ -1,2 +1,2 @@
 id,date_time,description,name,organization_uri,terms_of_use_uri,vip_id,version
-source01,2016-06-02T10:24:08,SBE is the official source for Virginia data,"State Board of Elections, Commonwealth of Virginia",http://www.sbe.virginia.gov/,http://example.com/terms,51,5.1
\ No newline at end of file
+source01,2016-06-02T10:24:08,SBE is the official source for Virginia data,State Board of Elections, Commonwealth of Virginia,http://www.sbe.virginia.gov/,http://example.com/terms,51,5.1
```
Comment (jdmgoogle) [inline docs/csv/example_files/source.txt]: You actually probably needed the quotes around the `name` column, since the value has a comma in it.

**Pull request merged**